lib/gui/etcher.js

```diff
@@ -21,16 +21,25 @@ const path = require('path')
 const _ = require('lodash')
 const { autoUpdater } = require('electron-updater')
 const Bluebird = require('bluebird')
+const semver = require('semver')
 const EXIT_CODES = require('../shared/exit-codes')
 const buildWindowMenu = require('./menu')
 const settings = require('./app/models/settings')
 const analytics = require('./app/modules/analytics')
 const { getConfig } = require('../shared/utils')
+const { version, packageType } = require('../../package.json')
 /* eslint-disable lodash/prefer-lodash-method */
+/* eslint-disable no-magic-numbers */
 
 const config = settings.getDefaults()
-
 const configUrl = settings.get('configUrl') || 'https://balena.io/etcher/static/config.json'
+const UPDATABLE_PACKAGE_TYPES = [
```

Comment:
Just out of curiosity, what's the rationale for some `const` variables being camelCase and others being UPPER_CASE ?

Reply:
Just a quick change without realising it's not ane env variable, going back to camel case in the next push